### DISCUSSION TITLE
DCDAO Steering Committee

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#47] [Ideas]
# Introduction

DCDAO is in its initial stages of formation, and the Steering Committee is appointed temporarily until a formal election process can be held. Members of this committee will regularly set up weekly meetings to review proposals and forum topics, coordinate and communicate resources, give suggestions for revision and improvement, and propose action guidelines. (Limit the number of members to <=5 to keep meetings efficient)

# Scope

1. Strategic Planning: Discussing and establishing the long-term goals, mission, and vision of the DCDAO.
2. Financial Support: Review the financial status, budget, and expenditures to ensure transparency and give suggestions respectively.
3. Governance Framework: Evaluating the governance structure and proposing changes or updates to ensure effective decision-making and participation from all stakeholders. Elect the DCDAO core members. 
4. Decision-Making: Making important decisions that impact the organization, considering input from various stakeholders, and ensuring that the decisions align with the organization's values and objectives in the early stage. 

# Term Limits and Rotation

Establish clear term limits for Steering Committee members to ensure regular turnover and prevent stagnation. Consider implementing a rotation policy to allow new members to bring fresh perspectives while ensuring continuity within the committee. 

The full term of DCDAO Steering is one year, and all DCDAO Steering Committee members are put up for election every June of every year.

# Committee Election

## Election Stage Overview

Process for selecting and voting for the new cohort of Steering Committee members.

1. Nominee selection (10 days). Candidates must gain 1/3 of the total voter support to make it to the next step.
2. Compliance check by DCDAO Community (10 days). The voice from the DCDAO community plays a critical role in fostering a collaborative and inclusive organizational culture, driving the non-profit DAO's mission forward, and ensuring that it remains responsive to the needs and expectations of its community members.
3. Member election (14 days). Votes are cast and the top 5 nominees are selected.

### 1. Nominee Selection (10 days)

Fill out your basic information and explain what’s your contribution in[ this form](https://forms.gle/PywQGkJjZesbi9Sv7) before 7/6 every year. The DCDAO community manager will review the application and publish the candidate list on 10/6. 

Nominee Qualification 

1. DCDAO NFT Holder 
2. Get involved in the DCDAO Governance business or contribute to DCDAO or Darwinia 

### 2. Compliance check by the DCDAO Community (10 days)

Seek input from the broader community of the DCDA, allowing members to provide feedback on the candidates and their suitability for the Steering Committee in the discussion. DCDAO ambassadors can serve as a bridge between the community and the Steering. In the case of doubt as to the candidate's qualifications, the community manager must check and give a reasonable reply within 3 days.

This process will be ended on 20/6.

### 3. Member Election(14 days)

The voting process will be held on [DCDAO SnapShot ](https://snapshot.org/#/darwiniacommunitydao.eth)during the 21/6-4/7. Every DCDAO NFT holder can take part in the election of the five representatives.  The top 5 nominees will be the new DCDAO Steering Committee on 5/7.

# Principle

1. Transparency. The Steering Committee needs to maintain transparency and open communication with the members of the non-profit DAO, as well as to uphold the principles of fairness, equity, and inclusivity in all decision-making processes. 
2. Regularly Scheduled Meeting. Regularly scheduled meetings can help ensure that the Steering Committee remains proactive in addressing the organization's challenges and opportunities, thereby fostering its growth and sustainability.

Click [here](https://dcdao.notion.site/d795187a20bd4beabc27e371b9699e86?v=ab263d64649d456b9840e501c4cb52e6&pvs=4) to learn DCDAO Steering Committee Meeting.

**Comment by hackfisher** [comment]
Requesting more details about the election process, reference:

1. https://twitter.com/arbitrum_cn/status/1680841013701607426
2. https://forum.arbitrum.foundation/t/aip-6-security-council-elections/15425

**Comment by SasoLithops** [comment]
Hi!

I am for Steering committee but would have quarterly votes for members who are part of this committee. One year seams a long time imho.

Reason why I am for it is in my understanding (will post more about it in another Github disscussion) even DAOs need some sort of "centralized" members( who are voted by community ) that push forward evolution of DAO and take care of technical,social media, realization and so on what they and community have proposed/discussed.

And to add strict rules must be applied for committee transparency. I would propose monthly report on what was done and spent for that month and share it with community. Also maybe AMA with community every month in Discord few days after report is out for questions they might have.

**Reply by Cindy884** [reply]
Thks for your feedback. What length of term do you think is reasonable?

**Reply by SasoLithops** [reply]
Absolutely, glad to participate in discussion💯 Maybe every quarter for Steering committee. For updating community monthly.

**Reply by Cindy884** [reply]
I think maybe every six months is better. The quarter is too frequent for the DCDAO and maybe it will lower the efficiency. For a start-up, efficiency is more important.:)

**Reply by SasoLithops** [reply]
My idea was if committee does well they will most probably ( in DAO I know not in politics😅 ) get all the votes and can continue with good work. But half a year seems reasonable for start as well.  

And I would propose that no member can be member of committee for more then 2 years to prevent DAO centralization interest. And community always has to vote on any committee decision regarding funds spending.

**Reply by Cindy884** [reply]
Thanks for the advice Saso. My view is still that since this committee is concerned with long-term strategic planning, it is not advisable to change it frequently. I have posted the proposal and you can vote on your view.

**Reply by SasoLithops** [reply]
My opinion was only on voting not changing them. 
As said if they do good they will get votes to continue their work😄

**Comment by hackfisher** [comment]
Current initial Committee disclosure(from initial 5 members):
1. fisher
2. Akira
3. Nada
4. Ran
5. Cindy

**Comment by SasoLithops** [comment]
I was having doubts about steering committee deciding who gets DCDAO NFT. However I have come to conclusion that most probably in order for things to work we do need time and some kind of "entity" in the begining to push things forward. I was hoping for it to be the code but if we vote on who members of steering committee are to lead the DCDAO I don't see any issues as long as this doesn't get political like I have seen in so many DAOs where certain people took over the DAO by their unisided views and stopped listening to community and responding to their questions alltogether. I know reaching consensus all the time is in practice impossible, but responding and participating in community discussions is a must in my opinion. Also keeping community updated on what is discussed in meetings so we can also add our opinions is much appreciated. We in this together.

Writing all that, I think it is fair for committee to have the initial 5 members from Darwinia since all the funding comes from Darwinia treasury.

**Reply by hackfisher** [reply]
I am good and glad that the election can be held earlier.

Nominee selection period may take more time than expected.

If there are enough nominees, I may not nominee myself for next year as I have too many other things to follow.( I will still give advice )

For the last/initial election, since there are only initial members in the set, and they are the only limited set for nominee to be elected, I didn't see any problem here. It is void before it was founded, not to mention making governance.
(Bitcoin is only mined by Satoshi before other miners join, it is also centralized at the starting point)

**Reply by SasoLithops** [reply]
I agree. 
Let me add that if voting was today I would vote for all five of you guys. I personaly have ABSOLUTE trust you have only good intentions for DC DAO.
